client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -1,207 +1,201 @@
-
-                        <!-- Password -->
-                        <div class="form-group" data-ng-class="{
-                            'has-error': ctrl.migrateDestinationForm.accountPassword.$dirty && ctrl.migrateDestinationForm.accountPassword.$invalid,
-                            'has-success': ctrl.migrateDestinationForm.accountPassword.$dirty && ctrl.migrateDestinationForm.accountPassword.$valid
-                        }">
-                            <label class="control-label" for="accountPassword" data-i18n-static="email_tab_modal_create_account_account_password"></label>
-                            
-                            <input type="password" class="form-control" id="accountPassword" name="accountPassword" aria-describedby="emailsChangePasswordHelp"
-                            autocomplete="off" required data-ng-maxlength="ctrl.constants.passwordMaxLength"
-                            data-ng-minlength="ctrl.constants.passwordMinLength" data-ng-model="ctrl.migrate.password">
-                            <small class="help-block" data-i18n-static="email_tab_modal_create_account_account_password_error" data-ng-show="ctrl.migrateDestinationForm.accountPassword.$dirty && ctrl.migrateDestinationForm.accountPassword.$invalid"></small>
-                            
-                        </div>
-
-                        <!-- Aknowledge Checkbox -->
-                        <div class="oui-checkbox">
-                            <input 
-                                type="checkbox" 
-                                id="aknowledgeCheckbox" 
-                                class="oui-checkbox__input" 
-                                data-ng-model="ctrl.migrate.aknowledge"
-                            >
-                            <label class="oui-checkbox__label-container" for="aknowledgeCheckbox">
-                                <span class="oui-checkbox__label">
-                                    <span data-ng-bind="ctrl.translator.tr('email_tab_modal_migrate_aknowledge')"></span>
-                                    <span class="oui-checkbox__icon">
-                                        <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
-                                        <i class="oui-icon oui-icon-checkbox-checked" aria-hidden="true"></i>
-                                        <i class="oui-icon oui-icon-checkbox-checkmark" aria-hidden="true"></i>
-                                        <i class="oui-icon oui-icon-checkbox-indeterminate" aria-hidden="true"></i>
-                                    </span>
-                                </span>
-                            </label>
-                        </div>
-                        
+            <!-- No Service -->
+            <div data-ng-if="!ctrl.availableServices">
+                <p data-i18n-static="email_tab_modal_migrate_no_email_service"></p>
+                <p>
```

Comment:
Same story, useful `<p>` and `rel` attribute ?

Reply:
Done